packages/playground/src/utils/types.ts

```diff
@@ -15,10 +15,37 @@ export interface IPublicConfig {
   domain?: string;
 }
 
+/**
+ * Represents a date object with day, month, and year properties.
+ *
+ * @interface Date
+ * @property {number} day - The day of the month (1-31).
+ * @property {number} month - The month of the year (1-12).
+ * @property {number} year - The year.
+ */
+export interface Date {
```

Comment:
Why is this needed ? https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Date